eng/pipelines/coreclr/runtime-nativeaot-outerloop.yml

```diff
@@ -0,0 +1,196 @@
+
+# This pipeline includes the Native AOT tests that we don't have resources to
+# run on every PR but that we want to still have test coverage, we run this
+# pipeline on a schedule and also developers can run it via /azp run command on
+# PRs. This pipeline permits us to have the main runtime pipeline run the same
+# platforms in PRs and Scheduled builds.
+
+# Setting batch to true, triggers one build at a time.
+# if there is a push while a build in progress, it will wait,
+# until the running build finishes, and produce a build with all the changes
+# that happened during the last build.
+trigger:
+  batch: true
+  branches:
+    include:
+    - release/*.*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - '**.md'
+    - eng/Version.Details.xml
+    - .devcontainer/*
+    - .github/*
+    - docs/*
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - THIRD-PARTY-NOTICES.TXT
+
+schedules:
+  - cron: "0 9,21 * * *" # run at 9:00 and 21:00 (UTC) which is 1:00 and 13:00 (PST).
+    displayName: Nativeaot-outerloop main schedule
+    branches:
+      include:
+      - main
+    always: false # run only if there were changes since the last successful scheduled run.
+
+
+#
+# CoreCLR NativeAOT release build and libraries tests
+# Only when CoreCLR or library is changed
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    platforms:
+    - windows_x64
+    - windows_arm64
+    - osx_x64
+    - osx_arm64
+    - linux_x64
+    - linux_arm64
+    - linux_musl_x64
+    jobParameters:
+      testGroup: innerloop
```

Comment:
Is the test group still `innerloop`?

Reply:
I think so. This is a terminology issue, but even though this pipeline is "outer-loop", the test group is "inner loop." I could be wrong about that, though.